docs/js_run_devserver.md

```diff
@@ -93,7 +93,7 @@ Only `data` files that change on incremental builds are synchronized when runnin
 | <a id="js_run_devserver-command"></a>command |  The devserver command to run.<br><br>For example, this could be the bin entry of an npm package that is included in data such as <code>./node_modules/.bin/next</code>.<br><br>Using the bin entry of next, for example, resolves issues with Next.js and React being found in multiple node_modules trees when next is run as an encapsulated <code>js_binary</code> tool.<br><br>Only one of <code>command</code> or <code>tool</code> may be specified.   | String | optional | <code>""</code> |
-| <a id="js_run_devserver-copy_data_to_bin"></a>copy_data_to_bin |  When True, data files and the entry_point file are copied to the Bazel output tree before being passed         as inputs to runfiles.<br><br>        Ideally, the default for this would be False as it is optimal, but there is a yet unresloved issue of ESM imports         skirting the node fs patches and escaping the sandbox: https://github.com/aspect-build/rules_js/issues/362.         This is hit in some test popular runners such as mocha, which use native <code>import()</code> statements         (https://github.com/aspect-build/rules_js/pull/353). <br><br>        A default of True will prevent program such as mocha from following symlinks into the source tree. They will         escape the sandbox but they will end up in the output tree where node_modules and other inputs required will be         available. With this in mind, the default will remain true until issue #362 is resolved.   | Boolean | optional | <code>True</code> |
+| <a id="js_run_devserver-copy_data_to_bin"></a>copy_data_to_bin |  When True, data files and the entry_point file are copied to the Bazel output tree before being passed         as inputs to runfiles.<br><br>        Defaults to True so that a js_binary with the default value is compatible with js_run_binary with run_from_execroot         set to True, the default there.<br><br>        Setting this to False is more optimal in terms of inputs, but there is a yet unresolved issue of ESM imports         skirting the node fs patches and escaping the sandbox: https://github.com/aspect-build/rules_js/issues/362.         This is hit in some test popular runners such as mocha, which use native <code>import()</code> statements         (https://github.com/aspect-build/rules_js/pull/353). When set to false, a program such as mocha that uses ESM         imports may escape the execroot by following symlinks into the source tree. When set to True, such a program         would escape the sandbox but they will end up in the output tree where node_modules and other inputs required         will be available.   | Boolean | optional | <code>True</code> |
```

Comment:
```suggestion
| <a id="js_run_devserver-copy_data_to_bin"></a>copy_data_to_bin |  When True, data files and the entry_point file are copied to the Bazel output tree before being passed         as inputs to runfiles.<br><br>        Defaults to True so that a js_binary with the default value is compatible with js_run_binary with run_from_execroot         set to True, the default there.<br><br>        Setting this to False is more optimal in terms of inputs, but there is a yet unresolved issue of ESM imports         skirting the node fs patches and escaping the sandbox: https://github.com/aspect-build/rules_js/issues/362.         This is hit in some popular test runners such as mocha, which use native <code>import()</code> statements         (https://github.com/aspect-build/rules_js/pull/353). When set to false, a program such as mocha that uses ESM         imports may escape the execroot by following symlinks into the source tree. When set to True, such a program         would escape the sandbox but they will end up in the output tree where node_modules and other inputs required         will be available.   | Boolean | optional | <code>True</code> |
```

Reply:
🍔